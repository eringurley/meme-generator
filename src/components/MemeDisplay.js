import React from 'react';
import PropTypes from 'prop-types';

function MemeDisplay({ header, imgURL, footer }) {
  return (
    <>
      <section>
        <p>{header}</p>
        <img src={imgURL}/>
        <p>{footer}</p>
      </section>
    </>
  );
}

MemeDisplay.propTypes = {
  header: PropTypes.string.isRequired,
  imgURL: PropTypes.string.isRequired,
  footer: PropTypes.string.isRequired
};

export default MemeDisplay;
