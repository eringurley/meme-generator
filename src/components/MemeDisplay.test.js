import React from 'react';
import { shallow } from 'enzyme';
import MemeDisplay from './MemeDisplay';

describe('MemeDisplay component', () => {
  it('renders MemeDisplay', () => {
    const wrapper = shallow(<MemeDisplay header="hi" imgURL="image" footer="footer"/>);
    expect(wrapper).toMatchSnapshot();
  });
});
