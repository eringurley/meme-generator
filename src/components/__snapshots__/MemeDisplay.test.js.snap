// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MemeDisplay component renders MemeDisplay 1`] = `
<Fragment>
  <section>
    <p>
      hi
    </p>
    <img
      src="image"
    />
    <p>
      footer
    </p>
  </section>
</Fragment>
`;
